betterforms/multiform.py

```diff
@@ -43,6 +81,34 @@ def __init__(self, data=None, files=None, *args, **kwargs):
             fargs, fkwargs = self.get_form_args_kwargs(key, args, kwargs)
             self.forms[key] = form_class(*fargs, **fkwargs)
 
+        self.fields = OrderedDict()
+        for key, form in self.forms.items():
+            for name, field in form.fields.items():
+                self.fields[(key, name)] = field
+
+        self.order_fields(self.field_order)
+
+    def order_fields(self, field_order):
+        if field_order is None:
+            return
+        fields = OrderedDict()
+
+        def add_field(key, name):
+            full_name = (key, name)
+            try:
+                fields[full_name] = self.fields.pop(full_name)
+            except KeyError:  # ignore unknown fields
+                print("unknown field", full_name)
```

Comment:
The django code just passes, I don't think we need to print.

https://github.com/django/django/blob/master/django/forms/forms.py#L124